avocado/utils/memory.py

```diff
@@ -278,3 +279,35 @@ def get_buddy_info(chunk_sizes, nodes="all", zones="all"):
             buddyinfo_dict[chunk_size] += int(chunk_info)
 
     return buddyinfo_dict
+
+
+def set_thp_value(value, feature):
+    """
+    Sets THP feature to a given value
+
+    :param value : Value to be set to feature
+    :type value : str
+    :param feature : Thp feature to set
+    :type feature : str
+    """
+    thp_path = '/sys/kernel/mm/transparent_hugepage/'
+    thp_feature_to_set = os.path.join(thp_path, feature)
+    cmd = 'echo %s > %s ' % (value, thp_feature_to_set)
+    process.system(cmd, shell=True)
```

Comment:
please use `ignore_status=True` as a pram in process.system if command fails and script will error out ?

also can we have check for if value is already set that time i think no need to run this command 
may be 
```

if value!=get_thp_value(feature):
            process.system(cmd,ignore_status=True, shell=True)
```